x-pack/plugins/ml/public/application/timeseriesexplorer/timeseriesexplorer_help_popover.tsx

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useState } from 'react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { HelpPopover, HelpPopoverButton } from '../components/help_popover/help_popover';
+
+export const TimeSeriesExplorerHelpPopover = () => {
+  const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+
+  return (
+    <HelpPopover
+      anchorPosition="upCenter"
+      button={
+        <HelpPopoverButton
+          onClick={() => {
+            setIsPopoverOpen(!isPopoverOpen);
+          }}
+        />
+      }
+      closePopover={() => setIsPopoverOpen(false)}
+      isOpen={isPopoverOpen}
+      title={i18n.translate('xpack.ml.timeSeriesExplorer.popoverTitle', {
+        defaultMessage: 'Single time series analysis',
+      })}
+    >
+      <p>
+        <FormattedMessage
+          id="xpack.ml.timeSeriesExplorer.popoverBasicExplanation"
+          defaultMessage="This chart illustrates the actual data values over time for a specific detector. You can examine an event by sliding the time selector and changing its length. For the most accurate view, set the zoom size to auto."
```

Comment:
I was wondering if it was worth adding something like `so that the interval at which points are plotted matches the bucket span of the job` to `For the most accurate view, set the zoom size to auto.` but maybe that is too much information?

Reply:
I tried a couple of versions of that sentence and it felt too low-level, especially in that first paragraph. If we need to revisit this decision, we can add it in a future PR.